### ISSUE TITLE
display what controller is used on the ghost page

### ISSUE THREAD

**Issue by willsigg** [stblr/mkw-sp#345]
![controllermockup](https://user-images.githubusercontent.com/43506664/179120065-81e1d08f-0f30-4aac-bb86-c165302a70a7.png)

basically just like the image attached. The ghost UI element might need to be made wider to fit the flag and the controller icon

**Comment by willsigg** [comment]
I made this mockup myself in photoshop, this isnt a real screenshot in game just in case anyone gets confused

**Comment by stblr** [comment]
I don't think we can make the buttons wider because of 4:3, but we can reduce the spacing between elements.
